playbook.yml

```diff
@@ -112,3 +114,41 @@
       with_items:
         - vim
         - emacs
+  post_tasks:
+    - name: restart machine
+      command: shutdown -r now "Ansible updates triggered" removes=/var/run/reboot-required
+      async: 0
+      poll: 0
+      ignore_errors: true
+
+    - name: waiting for server to come back
+      local_action: wait_for host={{ ansible_ssh_host }} port=22 search_regex=OpenSSH delay=10
+      sudo: false
+
+    - name: Sync DB
+      command: drush sql-sync --yes @gwayppl.prod.gwp @self chdir=/drupal/sites/gwp
+      sudo: false
+
+    - name: Update DB
+      command: drush updatedb --yes chdir=/drupal/sites/gwp
+      sudo: false
```

Comment:
Do we need the Update DB step to be automated? What if we are testing module updates?